kratos/python/add_dofs_to_python.cpp

```diff
@@ -59,6 +59,9 @@ void  AddDofsToPython(pybind11::module& m)
         .def("append", [](ModelPart::DofsArrayType &self, Dof<double> &value) {
             self.push_back(&value);
         })
+        .def("unique", [](ModelPart::DofsArrayType &self) {
```

Comment:
I have my reserves, your proposal is more pythonic, but then is not consistent with c++ methods and other exposed methods, in any case I agree

Reply:
True. In this case, I'd leave it like this to be consistent with the `append` export.